packages/grafana-prometheus/src/querybuilder/parsingUtils.test.ts

```diff
@@ -41,3 +47,69 @@ describe('getString', () => {
     expect(getString(replaced, tree.topNode)).toBe(expr);
   });
 });
+
+describe('replaceBuiltInVariables', () => {
```

Comment:
only suggestion would be to update this name, as the tests actually test both the `replace` and `return` functions